src/components/global/H1.js

```diff
@@ -1,4 +1,3 @@
-import React from "react"
```

Comment:
So when not using functional components, we don't need to  import React even though we are basically setting a component? did you test this?

Reply:
yes, you do not need React, I tested these changes